src/protocol.cpp

```diff
@@ -2884,3 +2890,9 @@ void CProtocol::PutStringUTF8OnStream ( CVector<uint8_t>& vecIn, int& iPos, cons
         PutValOnStream ( vecIn, iPos, static_cast<uint32_t> ( sStringUTF8[j] ), 1 );
     }
 }
+
+void CProtocol::PutCountryOnStream ( CVector<uint8_t>& vecIn, int& iPos, QLocale::Country eCountry )
```

Comment:
Same again, here, `CLocale::QtCountryToWireFormatCountry` should hide everything and return a `ushort`.

Reply:
Thanks. I've
- renamed `CLocale::QtCountryToWireFormatCountry` to `CLocale::QtCountryToWireFormatCountry`**`Code`** and updated it to take an `unsigned short` instead of `QLocale::Country`
- removed the obsolete, redundant casting in `CProtocol::PutCountryOnStream`